### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sahilda**
spelling fix.

### CODE REVIEW

File: mods/gen1/README.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ All moves but Swift and Bide (while on duration, not first hit) have a 1/256 cha
...
 Partial Trapping Moves
 ----------------------
-Partial trapping moves let eiher Pokémon switch but target will be unable to move for its duration.
+Partial trapping moves let either Pokémon switch but target will be unable to move for its duration.
 
 TODO. Glitches not implemented
 ------------------------------
-There are a couple of divisions by zero in the original game in the cartridge. Those have not been implemented.
\ No newline at end of file
+There are a couple of divisions by zero in the original game in the cartridge. Those have not been implemented.
```
Comment (Zarel) [inline mods/gen1/README.md]: Why does this line say it's been changed?

Comment (sahilda) [inline mods/gen1/README.md]: Not sure to be honest - I used the online github editor and it unnecessarily marked this line as a changed.

Comment (Zarel) [inline mods/gen1/README.md]: That doesn't mean nothing changed, it means something invisible has changed. Possibly a hidden character, possibly a line ending. I'm going to merge on the assumption that it's harmless.

Comment (xfix) [inline mods/gen1/README.md]: This is addition of extra line at end of file which is fine to add.

Comment (Zarel) [inline mods/gen1/README.md]: Huh, so they don't show the "no newline" symbol for this?

**Pull request merged**